geode-management/src/main/java/org/apache/geode/cache/RegionShortcut.java

```diff
@@ -16,9 +16,9 @@
 package org.apache.geode.cache;
 
 /**
- * Each enum represents a predefined {@link RegionAttributes} in a {@link Cache}. These enum values
- * can be used to create regions using a {@link RegionFactory} obtained by calling
- * {@link Cache#createRegionFactory(RegionShortcut)}.
+ * Each enum represents a predefined RegionAttributes in a Cache. These enum values
+ * can be used to create regions using a RegionFactory obtained by calling
+ * Cache#createRegionFactory(RegionShortcut).
```

Comment:
Please convert the {@link} tags to {@code} instead and then also replace the `#` with a `.` (period). That way it will still be formatted nicely despite losing the link.

Reply:
Sure thing. Thanks JD!